js/analytics.js

```diff
@@ -5,15 +5,26 @@
 var URI = require('urijs');
 var _ = require('underscore');
 
+function trackerExists() {
+  if (typeof ga !== 'undefined') {
+    return true;
+  }
+}
```

Comment:
... and as long as the consumer calls `init`, `ga` will always exist, so I'm not totally sure if there was a strong reason to check `ga` existing in the first place. Were you seeing something different at some point?